src/ingest-pipeline/airflow/dags/.airflowignore

```diff
@@ -8,4 +8,4 @@ workflow_map.yml
 resource_map.yml
 celery_queue_map.yml
 diagnostics
-export_and_backup
+/export_and_backup
```

Comment:
I think the leading / before export_and_backup is the wrong syntax- 'diagnostics' is a directory and it seems to work without a /, and what little documentation I found for .airflowignore files doesn't suggest a / .  Do you think the / is needed?

Reply:
Ooh, good point. OK, that's a good reason for the '/'.